backend/src/index.ts

```diff
@@ -175,10 +178,16 @@ class RequestInput {
     if (parseInt(content.exp, 10) <= Math.ceil(Date.now() / 1000)) {
       throw new Error(`JWT expired ${content.exp}`);
     }
+    const existingUser = await db.get<StoredUser>(`user:${content.sub}`, "json");
```

Comment:
Could you store the `user:${content.sub}` as userKey (since it's used for the db.set below).